src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -404,18 +413,34 @@ def do_kill_pydev_thread(self):
 
 
 def create_server_socket(host, port):
-    s = socket(AF_INET, SOCK_STREAM)
-    s.settimeout(None)
+    server = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP)
+    if IS_WINDOWS:
+        try:
+            from socket import SIO_LOOPBACK_FAST_PATH
+            server.ioctl(SIO_LOOPBACK_FAST_PATH, True)
```

Comment:
`ioctl` call shouldn't be inside `try`, since we're only interested in `ImportError` for that one constant being imported. It can go into `else:`.